testing/tests/DevExpress.exporter/exceljsParts/exceljs.pivotGrid.tests.js

```diff
@@ -165,6 +165,7 @@ QUnit.module('Scenarios', moduleConfig, () => {
         const ds = {
             fields: [
                 { area: 'row', dataField: 'row1', dataType: 'string' },
+                { area: 'column', dataField: 'col1', dataType: 'string' },
```

Comment:
imho we should not change existing tests in this way: there the [row: row1, data: count] config was tested and after this change this config is not tested.